_internal/utils/preload.ts

```diff
@@ -0,0 +1,30 @@
+import { Middleware, Key, BareFetcher } from '../types'
+import { serialize } from './serialize'
+
+const REQUEST = new Map<string, Promise<any>>()
+
+export const preload = (key_: Key, fetcher: BareFetcher<any>) => {
+  const req = fetcher(key_)
+  const key = serialize(key_)[0]
+  REQUEST.set(key, req)
+  return req
+}
+
+export const middleware: Middleware =
+  useSWRNext => (key_, fetcher_, config) => {
+    const fetcher =
+      fetcher_ == null
```

Comment:
support more falsy fetcher values like `false`/`undefined`/etc?

Reply:
@huozhi Thank you! 
Because of the following line, all falsy fetcher values would be `undefined`. But I've changed this to ignore all falsy values.
https://github.com/vercel/swr/blob/9263ca79f0747f9132fdcd0cf5eb0e47f7ee08a8/_internal/utils/resolve-args.ts#L27